samples/BasicTest.ipynb

```diff
@@ -0,0 +1,475 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "dc2af620-3f33-4153-a1c7-e05d8a2c9153",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metadata = [\n",
+    "    {\n",
+    "        \"user\": \"john\",\n",
+    "        \"age\": 18,\n",
+    "        \"job\": \"engineer\",\n",
+    "        \"credit_score\": \"high\",\n",
+    "    },\n",
+    "    {\n",
+    "        \"user\": \"derrick\",\n",
+    "        \"age\": 45,\n",
+    "        \"job\": \"doctor\",\n",
+    "        \"credit_score\": \"low\",\n",
+    "    },\n",
+    "    {\n",
+    "        \"user\": \"nancy\",\n",
+    "        \"age\": 94,\n",
+    "        \"job\": \"doctor\",\n",
+    "        \"credit_score\": \"high\",\n",
+    "    },\n",
+    "    {\n",
+    "        \"user\": \"tyler\",\n",
+    "        \"age\": 100,\n",
+    "        \"job\": \"engineer\",\n",
+    "        \"credit_score\": \"high\",\n",
+    "    },\n",
+    "    {\n",
+    "        \"user\": \"joe\",\n",
+    "        \"age\": 35,\n",
+    "        \"job\": \"dentist\",\n",
+    "        \"credit_score\": \"medium\",\n",
+    "    },\n",
+    "]\n",
+    "texts = [\"foo\", \"foo\", \"foo\", \"bar\", \"bar\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "8c5deed1-66d1-450a-ae2e-dd9b9baafd08",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from langchain.embeddings import BedrockEmbeddings\n",
+    "from langchain.llms.bedrock import Bedrock"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "ea2bd2a1-0823-4d36-bbcc-68959e15eee0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "embeddings = BedrockEmbeddings(\n",
+    "        credentials_profile_name=os.environ.get(\"BWB_PROFILE_NAME\"), #sets the profile name to use for AWS credentials (if not the default)\n",
```

Comment:
do we really need BWB_PROFILE_NAME, BWB_REGION_NAME and BWB_ENDPOINT_URL? AFAIK, these env. variables are only needed for Bedrock pre-GA relase. Since Bedrock GA is already released, we do not need them. Why not use the "default" profile?

Reply:
updated